package.json

```diff
@@ -20,13 +20,16 @@
   "homepage": "https://github.com/knight-18/Medical-Records#readme",
   "dependencies": {
     "bcryptjs": "^2.4.3",
+
     "check-password-strength": "^1.0.15",
```

Comment:
Is there really a need to use a separate package for checking password strength, that too in backend?

Reply:
Oh, I guess this is a mistake. I have forgotten to scrap that off package.json. Doing it now. 

There is no need for that package.